integration/subcommand_test.go

```diff
@@ -345,7 +345,7 @@ var _ = Describe("Subcommand", func() {
 		})
 	})
 
-	Describe("ginkgo blur", func() {
+	FDescribe("ginkgo blur", func() {
```

Comment:
Uh oh, looks like this focus is breaking the Travis build!

Reply:
Rha. Sorry, too hasty there.